src/plugins/intel_cpu/src/emitters/x64/jit_load_store_emitters.hpp

```diff
@@ -171,6 +178,11 @@ class jit_store_emitter : public jit_emitter {
     mutable bool data_reg_updated = false;
     mutable int data_idx = 0;
     mutable int aux_src_idx = 0;
+
+    size_t start_address = 0;
+    size_t current_address = 0;
+    size_t iteration = 0;
+    std::shared_ptr<snippets::op::Store> m_store_node;
```

Comment:
This is intend to print node name of blamed emitter, but not sure "store pointer of node in emitter" is a good method. 1. all the emitters need to be updated to have a pointer of node. 2. Lots of emitters do not have node in its' constructor, so can not set/get successfully.
Another option is to [save all expressions in generate](https://github.com/openvinotoolkit/openvino/pull/19935/files#diff-3c0cab6cbe926348e49ee79a66ea4c04056b8c2aff2d45b056814ae902176fbeR47-R52). After blamed emitter is set, search expressions to get the blamed expression, then get node and it's name? what do you think this way? thanks!

Reply:
The alternative option will not work in case of snippets `StoreEmitter` because it creates `jit_store_emitter` inside. If segfault happens during `jit_store_emitter` execution, `g_snippets_err_handler` returns its pointer whereas linear_ir contains `StoreEmitter`  pointer.
However, I don't think that it's good idea to save a pointer on Snippets node inside common cpu emitter.

Maybe we shouldn't bother with printing node name of blamed emitter? @IvanNovoselov @chenhu-wang what do you think?

Reply:
maybe node info is provided only for load store emitters, segfault is mostly happened here.

Reply:
Considering that we use the segfault detector only for snippets node, I would suggest to build debug info only in snippets emitters (Specifically in this case don't build info in jit_store_emitter, and do that only in StoreEmitter).

With this approach, we can save Store node in the snippets emitter and take the name from the node when building debug info

Reply:
Make sense.
1. build_debug_info and print_debug_info is done for parent emitters(snippets load/store/loadConvert/storeConvert emitters), which have nested underlying jit_load/jit_store emitter.
2. Underlying jit_load/jit_store emitter will not build_debug_info to avoid overide(never save to global handler).
3. In print_debug_info of snippets load/store/loadConvert/storeConvert emitters, underlying load/store emitters' print_debug_info is called to provide more detailed info(start address, current address, load num, iteration ,etc )
4. Node name is provided in print_debug_info of snippets load/store/loadConvert/storeConvert emitters.